README..md

```diff
@@ -0,0 +1,65 @@
+# the-covid-datacenter
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# The Covid Data Center
+
+This was our React capstone project for Microverse curiculum. It is mobile web application to check a list of metrics (numeric values) for the Covid-19 in United States of America that I created making use of React and Redux and the Coronavirus data API from Narrativa.com.
+
+![screenshot](./screenshot1.png)
+
+![screenshot](./screenshot2.png)
+
+## Built With
+
+- React js
+- Redux
+- React Bootstrap 
+- Bootstrap
+- Javascript
+- CSS
+- JSX
+
+## Live Demo
+[Live Demo](https://objective-pasteur-0b310b.netlify.app/)
+
+## Demo Video
+[Demo Video](https://www.loom.com/share/6b894b02e57d472cb6eb2375e188fa11)
+
+## Getting Started
+
+- Clone the repository with `git clone` https://github.com/DyaryRaoof/the-covid-datacenter.git;
+- Open the cloned folder on the IDE of your choice;
+- Install dependencies with `npm i`;
+- Use `npm start` to start the website;
+- Use `npm test` to run tests;
+- 
+## Authors
```

Comment:
- [ ] Kindly change the header title _Authors_ to _Author_, as this project is done solely.